src/meta/src/hummock/compaction/picker/compaction_task_validator.rs

```diff
@@ -64,6 +64,39 @@ impl CompactionTaskValidator {
         CompactionTaskValidator { validation_rules }
     }
 
+    pub fn unused(config: Arc<CompactionConfig>) -> Self {
+        let mut validation_rules: HashMap<
+            ValidationRuleType,
+            Box<dyn CompactionTaskValidationRule>,
+        > = HashMap::default();
+
+        validation_rules.insert(
+            ValidationRuleType::Tier,
+            Box::new(TierCompactionTaskValidationRule {
+                config: config.clone(),
+                enable: false,
+            }),
+        );
+
+        validation_rules.insert(
+            ValidationRuleType::Intra,
+            Box::new(IntraCompactionTaskValidationRule {
+                config: config.clone(),
+                enable: false,
+            }),
+        );
+
+        validation_rules.insert(
+            ValidationRuleType::ToBase,
+            Box::new(BaseCompactionTaskValidationRule {
+                config,
+                enable: false,
+            }),
+        );
```

Comment:
Why do we need to add these rules with enable=false? Will there be an issue if we use a task_validator with no rules?

Reply:
when `enable = false`, rule of task_validator will be skiped